src/main/java/jimmy/mcgymmy/model/ModelManager.java

```diff
@@ -142,6 +144,22 @@ private void addCurrentStateToHistory() {
         mcGymmyStack.push(new McGymmy(mcGymmy));
     }
 
+    @Override
+    public void clearFilteredFood() {
+
```

Comment:
Can you add `addCurrentStateToHistory();` so that undo command can refer to this state later